test/testing/test_utils.py

```diff
@@ -179,6 +180,14 @@ def wait_for_event(self):
 
     max_workers = DEFAULT_MAX_WORKERS
 
+    # TODO: pytest.warns is not supported until pytest >= 2.8.0, whose
```

Comment:
I intentionally duplicated them so we could find all the occurrences of the old pattern when a newer pytest is available